common/table.cpp

```diff
@@ -83,6 +83,48 @@ void Table::del(std::string key, std::string /* op */)
                            "DEL operation failed");
 }
 
+bool Table::getField(std::string key, std::string field, std::string &value)
+{
+    redisReply *reply = (redisReply*)redisCommand(
```

Comment:
Is it better to use RedisReply class to do the staffs? Similar to the previous Table::get function